app/services/witter.js

```diff
@@ -0,0 +1,3 @@
+const axios = require('axios');
+
+exports.getRandomQuote = () => axios.get(`${process.env.WITTER_API}/quotes/random`);
```

Comment:
Use [this config](https://github.com/wolox-training/fdr-express-js/blob/kickoff/config/index.js) for the environment variables